po/de.po

```diff
@@ -2118,11 +2118,10 @@ msgstr "Betrag"
 #. Translators: %d is the number of bills/credit notes due. This is a
 #. ngettext(3) message.
 #: ../gnucash/gnome/dialog-invoice.c:3435
-#, fuzzy, c-format
```

Comment:
Shouldn't the c-format indicator stay in this case as the plural forms contain a %d ? I am not that well aquainted with the po rules, so I may be mistaken as well.

Reply:
Good catch!

Yes, if we have at least one parameter '#, c-format' has to stay.